source/refunding_payments/index.html.md.erb

```diff
@@ -5,231 +5,106 @@ weight: 75
+Example response:
 
 ```javascript
 {
-"amount": 1500,
-"created_date": "2016-10-17T16:53:03.213Z",
-"refund_id": "j6se0f2o427g28g8yg3u3i",
-"status": "submitted",
-...
+  "amount": 2000,
+  "created_date": "2019-09-19T16:53:03.213Z",
+  "refund_id": "j6se0f2o427g28g8yg3u3i",
+  "status": "submitted",
+  ...
 }
 ```
 
-In a live environment, the status returned will initially be `submitted`.
-After the PSP has processed the refund, the status returned will be
-`success` or `error`. In a test environment, the status will always
-go directly to `success`.
-
-<div style="height:1px;font-size:1px;">&nbsp;</div>
-
-| Refund processing status | Meaning                                                                                                            |
-|--------------------------|--------------------------------------------------------------------------------------------------------------------|
-| submitted                | The refund request is valid as far as GOV.UK Pay can determine. It has been passed to the underlying payment service provider. |
-| success                  | The refund has been successfully processed.                                                                        |
-| error                    | It was not possible for the payment service provider to make the refund.                                                  |
-
-<div style="height:1px;font-size:1px;">&nbsp;</div>
-
-To manage refunds in live environments, you must use <a
-href="https://govukpay-api-browser.cloudapps.digital/#getrefundbyid"
-target="blank">Find payment refund by ID</a> to
-check the processing status of the refund, until it changes to either
-`success` or `error`.
-
-It will typically take 30 minutes for the status to change. You should not
-check the status more than once every 5 minutes.
-
-In the event of an error, GOV.UK Pay will not currently provide any more
-information. Please [contact
-us](/support_contact_and_more_information/#contact-us) if you need more
-information required about why a refund failed.
-
-## Refunding from the GOV.UK Pay admin tool
+If GOV.UK Pay could not successfully create the refund, the JSON response will contain an [API error code](https://docs.payments.service.gov.uk/api_reference/#errors-caused-by-an-api-call) and a description of the error.
 
-You can use the [GOV.UK Pay admin
-tool](https://selfservice.payments.service.gov.uk) to view transactions and
-issue refunds.
+## Checking the status of a refund
 
-Go to the __Transactions__ section to see a list of transactions. For example,
-in the following image::
+`GET /v1/refunds/PAYMENT-ID`
 
-![](/images/transaction+list+image+4.png)
+The JSON response will include a `status` field.
 
-In this list, you can select an individual payment to see details of that
-payment, including any previous refunds.
+The `status` usually changes every 30 minutes. Do not check the status more than once every 5 minutes.  
```

Comment:
Not sure about `The status usually changes every 30 minutes`?

Reply:
Happy to change this. Is there a more accurate figure? Or we could say "It may take up to X for the `status` to change" if that's preferable?

Reply:
Sorry this was a bit unclear. I think the intent of the sentence is "Typically changes to refund status take 30 minutes to be reflected in GOV.UK Pay reporting" or something? This is actually quite PSP dependent, with some PSPs it will be more or less instantaneous. I guess the main purpose of this guidance is to give services a ball park figure for how regularly to check for updates, so maybe something along the lines of ☝️ might work better?

Reply:
Ah I see, thanks! So I think "It may take up to 30 minutes for `status` to change" is probably the right approach? I do take the point that it misses a bit of the nuance - in that it's the change being reflected, rather than the change itself. But from the user's perspective, it should get the facts across clearly - which is that they shouldn't worry if they don't see the change immediately.

Reply:
yep, that sounds good